WTG.Analyzers/FutureSyntaxKinds.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace WTG.Analyzers
+{
+	static class FutureSyntaxKinds
+    {
+		const int DefaultLiteralExpression = 8755;
+
+		public static bool IsDefaultLiteralExpression(SyntaxKind kind) => kind == (SyntaxKind)DefaultLiteralExpression;
```

Comment:
I was thinking you would do something more like this:

    public const SyntaxKind DefaultLiteralExpression = (SyntaxKind)8775;

And then use it like you would any of the SyntaxKind members.